spark/src/test/scala/org/apache/spark/sql/CometTPCDSQuerySuite.scala

```diff
@@ -27,7 +27,7 @@ class CometTPCDSQuerySuite
     extends {
       // This is private in `TPCDSBase`.
       val excludedTpcdsQueries: Seq[String] =
-        Seq("q34", "q66", "q64", "q71", "q88", "q90", "q96")
+        Seq("q66", "q71", "q88", "q90", "q96")
```

Comment:
I just ran `CometTPCDSQuerySuite` locally and found two queries are working now.